src/libs/constants/libs/types/review-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+type ReviewDto = {
+	authorName: string;
+	content: string;
+	imageUrl: string;
+};
+
+export { type ReviewDto };
```

Comment:
types should not be inside of the constants.
move this type to the `home/libs/types/`